source/chapters/appendix.rst

```diff
@@ -443,7 +437,7 @@ Version History
   * Disable track metadata export for .ogg files and TagLib 1.11.1. `lp:1833190 <https://bugs.launchpad.net/mixxx/+bug/1833190>`__
   * Fix interaction of hot cue buttons and looping. `lp:1778246 <https://bugs.launchpad.net/mixxx/+bug/1778246>`__
   * Fix detection of moved tracks. `#2197 <https://github.com/mixxxdj/mixxx/pull/2197>`__
-  * Fix playlist import. `lp:1687828 <https://bugs.launchpad.net/mixxx/+bug/1687828>`__
+  * Fix playlist import. `lp:16878282 <https://bugs.launchpad.net/mixxx/+bug/16878282>`__
```

Comment:
please revert this, that link is broken now

Reply:
Ok, will do this!